controller/app.controller.js

```diff
@@ -9,3 +9,15 @@ exports.getTopics = (req, res, next) => {
       next(err);
     });
 };
+
+exports.getArticles = (req, res, next) => {
```

Comment:
I think this controller name is a bit vague. 'getArticleByarticleId' - would make more sense